react-basic-practice/src/constants/modal.ts

```diff
@@ -4,3 +4,7 @@ export const MODAL_TITLES = {
   VIEW_CUSTOMER: "View Customer",
   DELETE_CUSTOMER: "Delete Customer",
 };
+
+export const MODAL_DESCRIPTION = {
+  CONFIRM_DESCRIPTION: "Are you sure you want to delete this customer?",
```

Comment:
it's should be CONFIRM_DELETE_DESCRIPTION

Reply:
i done